src/cfn/estimateStackCost.ts

```diff
@@ -0,0 +1,15 @@
+import {SUCCESS} from '../statusCodes';
```

Comment:
I think this filename should be `TitleCased` (`EstimateStackCost`) given that it is a class.